src/containers/PageBuilder/SectionBuilder/SectionBuilder.module.css

```diff
@@ -181,7 +181,9 @@
 }
 
 /* link on dark theme */
-.darkTheme a {
+
+/* ************************************ */
+.darkTheme .Link__pages {
```

Comment:
Did you check if this approach actually works after our short chat about the issue? 😉 

It does not: the class is not used at all because this creates CSS Modules class, which is not passed to the component.
In CSS Modules, global classes needs to be defined with `global:` scope:
https://github.com/css-modules/css-modules/blob/master/docs/composition.md#exceptions

That being said, I think we need to change this to work properly with CSS Modules. I.e. pass this generated class from SectionBuilder to Link. They key for that is to turn [this variable](https://github.com/sharetribe/web-template/blob/main/src/containers/PageBuilder/Field/Field.js#L63-L110) into a function (and pass the class to fields.js).